hack/feature-deploy.sh

```diff
@@ -45,16 +52,21 @@ do
   done
 
   if [[ $feature_failed -eq 1 ]]; then
+
     iterations=$((iterations + 1))
     iterations_left=$((max_iterations - iterations))
-    echo "[WARN] At least one deployment failed, retrying in $sleep_time sec, $iterations_left retries left"
-    sleep $sleep_time
-    continue
+    if [[ $iterations_left != 0  ]]; then
+      echo "[WARN] At least one deployment failed, retrying in $sleep_time sec, $iterations_left retries left"
```

Comment:
Just to not scare customers, should we change the like here to - The deployment is not fully ready, retrying... ?

Reply:
ok, done